### PR TITLE
Translate the pitch

### PR DESCRIPTION
**Pull request by kantoniak**
Translate the initial pitch into English. Extended on benefits and shortened the technical part.

### CODE REVIEW

File: specs/pitch.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# MyCanteen
+
+## Problem statement
+
+Cafeterias are popular among students and office workers going for lunch or dinner. Since people tend to eat at the same time, lunchrooms are usually cramped. Customers have to wait in longer queues and look for empty seats afterwards. Also, visitors have to come to cafeteria to find out what is served, how long will they wait etc. Owners have to handle customer spikes during lunchtime while cafes are empty at different hours.
```
Comment (avan1235) [inline specs/pitch.md]: longer => long || longer and longer
why do you call these places as cafes/cafeterias?

File: specs/pitch.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Value proposition
+
+Informing customers about cafeteria state saves them time and helps them decide whether to go for lunch. They should know beforehands the menu and how many people are currently having lunch. With rating options, users can guide each other in search for the best dishes. Foreign visitors and disabled users can easily access all data on their devices.
```
Comment (avan1235) [inline specs/pitch.md]: Cafeteria again ;)

File: specs/pitch.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Owners will benefit from analytics. Historical data and occupancy predictions help to prepare for lunch in advance, while updating customers mitigates traffic spikes. Publishing menus is a new marketing channel promoting cafe directly to interested users. Administration panel can automate posting to social media channels. Collected feedback will contribute to service improvements.
+
+## Architecture overview
+
+System of cameras counts visitors entering and leaving lunchroom space. Central endpoint aggregates occupancy reports for owners and customers using mobile or web clients, which also serve as information/feedback loop.
```
Comment (avan1235) [inline specs/pitch.md]: comma before which only in polish xd

Comment (kantoniak) [reply specs/pitch.md]: Fixed.

File: specs/pitch.md (hunk 1 above)
Comment (kantoniak) [reply specs/pitch.md]: Fixed.
These are the same, I think it may be British vs American English thing.

File: specs/pitch.md (hunk 2 above)
Comment (kantoniak) [reply specs/pitch.md]: Same as up

**Pull request merged**